llama_index/vector_stores/postgres.py

```diff
@@ -101,75 +112,108 @@ def _create_extension(self) -> None:
                 session.execute(statement)
                 session.commit()
 
+    def _node_to_table_row(self, node: NodeWithEmbedding) -> Any:
+        return self.table_class(
+            node_id=node.id,
+            embedding=node.embedding,
+            text=node.node.get_content(metadata_mode=MetadataMode.NONE),
+            metadata_=node_to_metadata_dict(
+                node.node,
+                remove_text=True,
+                flat_metadata=self.flat_metadata,
+            ),
+        )
+
     def add(self, embedding_results: List[NodeWithEmbedding]) -> List[str]:
         ids = []
         with self._session() as session:
             with session.begin():
                 for result in embedding_results:
                     ids.append(result.id)
-
-                    item = self.table_class(
-                        node_id=result.id,
-                        embedding=result.embedding,
-                        text=result.node.get_content(metadata_mode=MetadataMode.NONE),
-                        metadata_=node_to_metadata_dict(
-                            result.node,
-                            remove_text=True,
-                            flat_metadata=self.flat_metadata,
-                        ),
-                    )
+                    item = self._node_to_table_row(result)
                     session.add(item)
                 session.commit()
         return ids
 
+    async def aadd(self, embedding_results: List[NodeWithEmbedding]) -> List[str]:
```

Comment:
Does `aadd` sound as awkward to you as it does to me? I was trying to follow the existing naming convention with the way we're distinguishing sync vs. async functions but I wouldn't mind renaming this to `async_add`

Reply:
Ha I left the same comment as I was going through this.

I think async_add can be an exception to our naming scheme, aadd just looks like a typo imo